features/forced-colors.yml

```
@@ -0,0 +1,14 @@
+name: Forced colors
+description: The `forced-colors` CSS @media rule and `forced-color-adjust` CSS property control styles when a user has chosen a limited palette of colors in forced colors mode.
```

```suggestion
description: The `forced-colors` CSS @media rule detects when a user has chosen to use a forced colors mode, also known as high-contrast mode, and the `forced-color-adjust` CSS property sets whether force colors apply to an element.
```

Not sure if you like this better or not, but I feel like at least mentioning the term "high contrast" is necessary here. And splitting the 2 entry points into 2 sentences allows to give more details.